### PR TITLE
readme: Add DC/OS login

### PR DESCRIPTION
**Pull request by nlsun**

### CODE REVIEW

**Comment by nlsun** [comment]
@karlkfi please review

File: README.md [hunk 1]
```diff
@@ -107,6 +107,13 @@ Host Docker storage drivers of `overlay` and `aufs` are supported automatically.
 
 For other make commands, see `make help`.
 
+## DC/OS Login
+
```
Comment (karlkfi) [inline README.md]: please add a line about this being specific to Enterprise DC/OS installs and that OSS DC/OS uses OAuth for login.

**Comment by nlsun** [comment]
@karlkfi fixed

**Comment by karlkfi** [comment]
thanks!

**Pull request merged**